[2024-08-05T09:21:35.205+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T09:21:35.226+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Tokyo scheduled__2024-08-05T02:00:00+00:00 [queued]>
[2024-08-05T09:21:35.233+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Tokyo scheduled__2024-08-05T02:00:00+00:00 [queued]>
[2024-08-05T09:21:35.233+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-05T09:21:35.248+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_air_pollution_data_Tokyo> on 2024-08-05 02:00:00+00:00
[2024-08-05T09:21:35.252+0300] {standard_task_runner.py:64} INFO - Started process 21562 to run task
[2024-08-05T09:21:35.255+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'air_pollution_data_dag', 'fetch_air_pollution_data_Tokyo', 'scheduled__2024-08-05T02:00:00+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/air_pollution_data_dag.py', '--cfg-path', '/tmp/tmpjpu5hcwk']
[2024-08-05T09:21:35.255+0300] {standard_task_runner.py:91} INFO - Job 165: Subtask fetch_air_pollution_data_Tokyo
[2024-08-05T09:21:35.290+0300] {task_command.py:426} INFO - Running <TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Tokyo scheduled__2024-08-05T02:00:00+00:00 [running]> on host std21092
[2024-08-05T09:21:35.357+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='air_pollution_data_dag' AIRFLOW_CTX_TASK_ID='fetch_air_pollution_data_Tokyo' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-05T02:00:00+00:00'
[2024-08-05T09:21:35.357+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T09:21:40.681+0300] {logging_mixin.py:188} INFO - Erreur lors de la récupération des données pour Tokyo
[2024-08-05T09:21:40.682+0300] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T09:21:40.682+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T09:21:40.689+0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=air_pollution_data_dag, task_id=fetch_air_pollution_data_Tokyo, run_id=scheduled__2024-08-05T02:00:00+00:00, execution_date=20240805T020000, start_date=20240805T062135, end_date=20240805T062140
[2024-08-05T09:21:40.918+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T09:21:40.955+0300] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T09:21:40.959+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
